This patch makes setBreakType public (needed for OO.org),
pending a better upstream solution in 4.0. For details, see:
http://qa.openoffice.org/issues/show_bug.cgi?id=83146
http://bugs.icu-project.org/trac/ticket/5498
http://bugs.gentoo.org/show_bug.cgi?id=200010
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=448745
diff -Nau common/unicode.rbbi.h.orig common/unicode/rbbi.h
--- common/unicode/rbbi.h.orig	2007-10-31 15:52:08.000000000 +0100
+++ common/unicode/rbbi.h	2007-10-31 15:52:47.000000000 +0100
@@ -611,12 +611,14 @@
     virtual int32_t getBreakType() const;
 #endif
 
+public:
     /**
       * Set the type of the break iterator.
       * @internal
       */
     virtual void setBreakType(int32_t type);
 
+protected:
     /**
       * Common initialization function, used by constructors and bufferClone.
       *   (Also used by DictionaryBasedBreakIterator::createBufferClone().)
